backend/clubs/views.py

```diff
@@ -4831,16 +4852,58 @@ def initiate_checkout(self, request, *args, **kwargs):
                     reason=f"Received {context} with HTTP status {status}",
                 )
 
-            return Response({"success": True, "detail": context})
+            return Response(
+                {
+                    "success": True,
+                    "detail": context,
+                    "solf_free_tickets": False,
+                }
+            )
+
         except ApiException as e:
             return Response(
                 {
                     "success": False,
                     "detail": f"Unable to generate capture context: {e}",
+                    "sold_free_tickets": False,
                 },
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
+    def give_tickets(self, orderInfo, cart, reconciliation_id):
```

Comment:
nit: `order_info` (snake case)

Can you also move this helper below all other views in the viewset for consistency?